micro-rdk/src/native/entry.rs

```diff
@@ -30,8 +28,9 @@ pub async fn serve_web_inner(
     tls_server_config: NativeTlsServerConfig,
     repr: RobotRepresentation,
     ip: Ipv4Addr,
-    exec: NativeExecutor<'_>,
+    exec: NativeExecutor,
 ) {
+    log::info!("web start");
```

Comment:
Leftover debugging?, and on line 45 too.

Reply:
:P